src/Components/Spacer/style.module.scss

```diff
@@ -0,0 +1,9 @@
+@import "../../common/util";
+
+.halfWidth {
+  width: 75%;
```

Comment:
I'm assuming there's a reason for this, but... 🤔

Reply:
lol, too many constant updates from artists made me lose my mind, good catch